### DISCUSSION TITLE
blog/typora-notion-to-obsidian/

### DISCUSSION THREAD

**Discussion by giscus** [rootsongjc/rootsongjc.github.io#358] [Announcements]
# blog/typora-notion-to-obsidian/

探索我从 Typora 和 Notion 迁移到 Obsidian 的原因，深入对比三者优缺点，分享最佳实践。

https://jimmysong.io/blog/typora-notion-to-obsidian/

**Comment by xcbeyond** [comment]
可以直接管理其他类型的文件吗，比如word、Excel等

**Reply by rootsongjc** [reply]
虽然也可以将 Obsidian 当做一个文件管理器，但是它并不支持编辑非 Markdown 格式。

**Comment by gotoworld** [comment]
为知 --> Typora --> Obsidian  --> SiYuan（轻量，本地化）

另一位工程师的，笔记成长路径哈

**Reply by rootsongjc** [reply]
SiYuan UI 看起来跟 Notion 好像，不过功能上跟 Obsidian 有区别吗？

**Reply by Hugh-yw** [reply]
Typora ---> CSDN ---> 语雀 ---> Notion --->SiYuan ✍🏻
